plugins/models/cnn.py

```diff
@@ -0,0 +1,41 @@
+import keras
+from keras.layers import *
+from keras.models import Model, Sequential
+from keras.optimizers import Adam
+
+from deepprofiler.learning.model import DeepProfilerModel
+
+
+def define_model(config, dset):
```

Comment:
Is this the VGG architecture? Add some comments to the class definition and also a link to the paper if possible. A short version of this example: https://github.com/tensorflow/tensorflow/blob/4665633c5f3fcd98de8c4cab909c8ac64c28ed0f/tensorflow/contrib/slim/python/slim/nets/vgg.py#L15